docs/guides/endgame.md

```diff
@@ -921,6 +932,49 @@ the following testing scenarios:
 - [ ] React smoke tests
   - [ ] `create-react-app`
 
+<details>
+  <summary>How to run React smoke test</summary>
+
+Change directories to wherever you want to add the project and run:
+
+```bash
+npx create-react-app tmp
+```
+
+Change directories so you're in the newly created `tmp` folder and run `yarn` in
+your terminal.
+
+After that is done, install the dependencies: carbon-components,
+carbon-components-react, carbon-icons and node-sass (two dependencies that the
+other packages require) by running:
+
+```bash
+yarn add carbon-components@next carbon-components-react@next @carbon/icons-react
+// And
+yarn add node-sass
+```
+
+Once that's done, open up `tmp` in your code editor and change any `.css` files
+to `.scss` and add:
+
+```javascript
+@import 'carbon-components/scss/globals/scss/styles.scss';
+```
+
+to the top of `index.scss`.
+
+Update the `.css` imports to `.scss` in `App.js` and `index.js`. At the top of
+`App.js`, import the `Button` component by adding the following:
+
+```javascript
+import { Button } from 'carbon-components-react';
+```
+
+In the `App` component return, add the `Button` and then run `yarn start` and
+verify that the `Button` is there and looks correct.
```

Comment:
Should we add a note like "and no errors are logged to the console"?

Reply:
Added!